src/App.js

```diff
@@ -88,8 +88,13 @@ class Board extends React.Component {
 
   //Checks 3 by 3 block
   isPossibleBlock(randomNumber, position, valueBoard) {
+    let oneOfTheNumberInThreeByThreeBlock = 0;
```

Comment:
No need to declare this outside of the for loop, you can just declare it directly within.

Reply:
Done